.github/workflows/tools_tests.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install pytest pytest_mock
-          pip install azure-identity azure-keyvault-secrets
+          pip install azure-identity azure-keyvault-secrets azure-ai-ml          
```

Comment:
why we need `azure-ai-ml` here? I don't see any other change in this PR, could you please clarify the place where it requires this pkg?

Reply:
seems you need this in #976 , in that case, I think you better make this change in that PR and close this one

Reply:
this is needed as part of this PR https://github.com/microsoft/promptflow/pull/976.
I tried updatating tools_tests.yml as part of the above PR seems including it there had no effect on how the CI is setup.

see the linked comment here - https://github.com/microsoft/promptflow/pull/976#discussion_r1379617426